tests/bluetooth/controller/mock_ctrl/CMakeLists.txt

```
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# CMakeLists.txt file for creating of mocks library.
+#
+
+add_library(mocks STATIC
+  src/kernel.c
+  src/ecb.c
+  src/mayfly.c
+  src/lll.c
+  src/lll_conn.c
+  src/ll_assert.c
+  src/assert.c
+  src/util.c
+  src/ticker.c
+  src/ull.c
+  src/ull_conn_iso.c
+  src/ull_peripheral.c
+  src/ull_peripheral_iso.c
+  src/ull_central.c
+  src/ull_scan.c
+  src/lll_clock.c
+)
+
+
+target_include_directories(mocks PUBLIC
+  include
+  ${ZEPHYR_BASE}/subsys/bluetooth/controller/include
```

as you have decided that test lib themselves should not inherit includes under `${ZEPHYR_BASE}/subsys/bluetooth`, then those should be made `PRIVATE` as to avoid mix of principles.

```suggestion
  PRIVATE
  ${ZEPHYR_BASE}/subsys/bluetooth/controller/include
```